docs/content/how-to/disaster-recovery/backup-and-restore-oadp-1-5.md

```diff
@@ -0,0 +1,886 @@
+            objectStorage:
+              bucket: oadp-backup
+              prefix: hcp
+            config:
+              region: minio
+              profile: "default"
+              s3ForcePathStyle: "true"
+              s3Url: "http://registry.hypershiftbm.lab:9002"
+              insecureSkipTLSVerify: "true"
+            credential:
+              key: cloud
+              name: cloud-credentials
+              default: true
+      snapshotLocations:
+        - velero:
+            provider: aws
+            config:
+              region: minio
+              profile: "default"
+            credential:
+              key: cloud
+              name: cloud-credentials
+      configuration:
+        nodeAgent:
+          enable: true
+          uploaderType: kopia
+        velero:
+          defaultPlugins:
+            - openshift
+            - aws
+            - csi
+            - hypershift
+          resourceTimeout: 2h
+    ```
+
+=== "**Openstack**"
+
+    ```yaml
+    ---
+    apiVersion: oadp.openshift.io/v1alpha1
+    kind: DataProtectionApplication
+    metadata:
+      name: dpa-instance
+      namespace: openshift-adp
+    spec:
+      backupLocations:
+        - name: default
+          velero:
+            provider: aws
+            default: true
+            objectStorage:
+              bucket: example-oadp
+              prefix: backup-objects
+            config:
+              region: region-one
+              profile: "default"
+            credential:
+              key: cloud
+              name: cloud-credentials
+      snapshotLocations:
+        - velero:
+            provider: aws
+            config:
+              region: minio
+              profile: "default"
+            credential:
+              key: cloud
+              name: cloud-credentials
+      configuration:
+        nodeAgent:
+          enable: true
+          uploaderType: kopia
+        velero:
+          defaultPlugins:
+            - openshift
+            - aws
+            - csi
+            - hypershift
+          resourceTimeout: 2h
+    ```
+
+
+Once you create any of these DPA objects, several pods will be instantiated in the `openshift-adp` namespace. This includes one `node-agent` per node in the Management Cluster and the `velero` deployment.
+
+!!! note
+
+    To follow backup and restore procedures, you can monitor the logs in the velero pod.
+
+
+## Backup and Upload
+=== "**AWS**"
+    ### Data Plane workloads backup
+
+    !!! Note
+
+        If the workloads in the Data Plane are not crucial for you, it's safe to skip this step.
+
+    If you need to backup the applications running under the HostedCluster, it's advisable to follow [the official documentation for backup and restore of OpenShift applications](https://docs.openshift.com/container-platform/latest/backup_and_restore/application_backup_and_restore/backing_up_and_restoring/backing-up-applications.html)
+
+    The steps will indeed be quite similar:
```

Comment:
Not clear to what they are similar, I would just have something like "The steps are the following:"

Reply:
(same for the other 2 occurrences of this sentence below)